pylops/jaxoperator.py

```diff
@@ -0,0 +1,102 @@
+__all__ = [
+    "JaxOperator",
+]
+
+from typing import Any, NewType
+
+from pylops import LinearOperator
+from pylops.utils import deps
+
+if deps.jax_enabled:
+    import jax
+    import jaxlib
+
+    jaxarray_type = jaxlib.xla_extension.ArrayImpl
```

Comment:
My understanding is that jax should be typed with `jax.Array` or `jax.typing.ArrayLike`: https://jax.readthedocs.io/en/latest/jax.typing.html

Reply:
Sorry, not sure why I used that one... fixed